argus/scenarios/cloud.py

```diff
@@ -48,6 +51,81 @@ def get_remote_client(self, username=None, password=None,
     remote_client = util.cached_property(get_remote_client, 'remote_client')
 
 
+class NetworkWindowsScenario(BaseWindowsScenario):
+    """Scenario for testing static network configuration.
+
+    Creates an additional internal network which will be
+    bound explicitly with the new created instance.
+    """
+
+    def _get_isolated_network(self):
+        """Returns the network itself from the isolated network resources.
+
+        This works only with the isolated credentials and
+        this step is achieved by allowing/forcing tenant isolation.
+        """
+        # Extract the just created private network.
+        resources = self._isolated_creds.isolated_net_resources["primary"]
+        return resources[0]
+
+    def _fill_networks(self):

```

Comment:
Just define a method, _get_networks, which returns the networks you desire, without setting self._networks to something.